showcase/app/templates/components/app-header/index.hbs

```diff
@@ -243,13 +244,84 @@
 
   <Shw::Divider />
 
+  <Shw::Text::H2>Text</Shw::Text::H2>
+
+  <Shw::Grid @columns={{1}} {{style gap="2rem"}} as |SG|>
+
+    <SG.Item>
```

Comment:
As discussed, consider adding another example, similar to this one, only with an extra custom "badge" next to it, to demonstrate how consumers _could_ implement something that grabs the attention, really stands out

Reply:
Added an example of this